draft-ietf-jsonpath-base.md

```diff
@@ -419,8 +418,10 @@ constant.
 | `//book[position()<3]` | `$..book[0,1]`<br>`$..book[:2]`           | the first two books                                          |
 | `//book[isbn]`         | `$..book[?(@.isbn)]`                      | filter all books with isbn number                            |
 | `//book[price<10]`     | `$..book[?(@.price<10)]`                  | filter all books cheaper than 10                             |
-| `//*`                  | `$..*`                                    | all elements in XML document; all members of JSON data item  |
-{: #tbl-example title="Example JSONPath expressions applied to the example JSON data item"}
+| `//*`                  | `$..*`                                    | all elements in XML document; all member values contained in JSON value |
+{: #tbl-example title="Example JSONPath expressions applied to the example JSON value"}
+
+<!-- XXX: is $..* really just member values -->
```

Comment:
There is a [consensus](https://cburgmer.github.io/json-path-comparison/results/dot_notation_with_wildcard_after_recursive_descent.html) around `..*`: it traverses descendants (including the starting node) and for each one applies `.*` to objects and `[*]` to arrays.

Let's defer fine-tuning the description until a PR which covers `..` in detail.

Reply:
Right.  To reduce confusion, I did add "array elements" to the text in the table; but clearly, this needs to be addressed in a technical PR.